### ISSUE TITLE
Round subject marks

### ISSUE THREAD

**Issue by M4R1KU** [outcobra/outstanding-cobra#350]
Normally the subject marks will be rounded to 1/10th or 1/2th or other (depending on school). Currently we do calculate our marks very precisely in terms of math but it does not really reflect the real world. 
Example:
In the BMS-report I only have a 5 but Outcobra is telling me that I had a 5.08. The difference is very small but if you want to show your marks to someone like your coach you have to explain why it is different for example.

The user should be able to choose how the average marks should be calculated. 

Either on marks ui or in the subject create/update dialog.

**Comment by needToRoll** [comment]
I totally agree with you that this feature schould be implemented. Taking into account that there are other features that might need a possibility to customize specific behaviour of a feature I thinkl we might have to start talking about a settings page.
